modules/abstract-eth/src/abstractEthLikeCoin.ts

```diff
@@ -36,12 +37,12 @@ export interface TxInfo {
 }
 
 interface TransactionPrebuild extends BaseTransactionPrebuild {
-  txHex: string;
+  txHex?: string;
   txInfo: TxInfo;
   feeInfo: EthTransactionFee;
   source: string;
   dataToSign: string;
-  nextContractSequenceId?: string;
+  nextContractSequenceId?: number;
```

Comment:
BREAKING CHANGE: Type of nextContractSequenceId is changed to number to avoid typescript error of incompatible function params for signTransaction method

Reply:
Have created a ticket WIN-1098 to address the change required in OVC because of this